app/services/api.rb

```diff
@@ -0,0 +1,12 @@
+class Api
+  include HTTParty
+
+  def initialize
+    self.class.base_uri 'localhost:3001'
```

Comment:
we generally dont hardcode urls into our apps like this.
what we usually do is use a tool like https://github.com/bkeepers/dotenv to give ourselves environment variable files that hold this information that then isnt commited to the repo.

as these urls should be easy to change without needing to change the code